index.html

```diff
@@ -294,6 +294,7 @@
     <p>Source available on <a href="https://github.com/jrconlin/WebPushDataTestPage">GitHub</a>. Licensed under <a href="https://www.mozilla.org/en-US/MPL/2.0/">Mozilla Public License v.2.0</a></p>
 </div>
   </body>
+  <script src="common.js"></script>
   <script src="base64.js"></script>
   <script src="webpush.js"></script>
   <script src="vapid.js"></script>
```

Comment:
Yeah, I'm debating that. It's possible using git tricks to import that library as a dependency, but it's not pretty, at all. I really wanted to make this page as self contained as possible while still providing some level of reuse. 

I'm a complete noob when it comes to js packaging, so feel free to offer whatever guidance you want.